### PR TITLE
resolving #158

### PR DESCRIPTION
**Pull request by pvcodes**
Hi, I resolved issue #158. @eddiejaoude please give it a green flag.

### CODE REVIEW

**Comment by pvcodes** [comment]
> Hi, I resolved issue #158. @eddiejaoude please give it a green flag.

due to my "format document" of VSCODE, the App.svelte is formatted and that's why this much of insertion and deletion are showing

**Comment by pvcodes** [comment]
> # Alex Recommends Report
> 
> Alex recommends the following language changes, but Alex is a regular expression based algorithm, so take them with a grain of salt.
> ### sparkles rocket sparkles Nothing to Report sparkles rocket sparkles

Hi, @AllanRegush, this github report bot is soo overwhelming! :smile:

**Comment by AllanRegush** [comment]
> > # Alex Recommends Report
> > Alex recommends the following language changes, but Alex is a regular expression based algorithm, so take them with a grain of salt.
> > ### sparkles rocket sparkles Nothing to Report sparkles rocket sparkles
> 
> Hi, @AllanRegush, this github report bot is soo overwhelming! smile

Sorry I didn't see this tag. Yeah, there are a lot of messages.

**Comment by AllanRegush** [comment]
I'll have to pull this down tomorrow and have a look :+1:

**Comment by eddiejaoude** [comment]
hi @pvcodes , sorry for the slow reply, I am still going through GitHub notifications. Thank you for the changes, someone will take a look soon 👍 

Note, smaller PRs get merged sooner, because they are easier to review. If there are reformatting changes, it is best to do those in a separate PR

File: src/App.svelte [hunk 1]
```diff
@@ -37,23 +36,27 @@
     </p>
   </div>
 
+  <div>
+    <img
+      src="https://user-images.githubusercontent.com/76584243/111971513-6edc1c80-8b22-11eb-83df-277a0a8f4fd8.png"
+      alt="" />
```
Comment (schmelto) [inline src/App.svelte:42]:
```suggestion
      alt="EddieHub Banner" />
```
Maybe add some alternative text here

**Review by schmelto — CHANGES_REQUESTED** [review]

File: src/App.svelte (hunk 1 above)
Comment (pvcodes) [reply src/App.svelte]: Done!

File: src/App.svelte [hunk 2]
```diff
@@ -3,29 +3,28 @@
...
+  class="min-w-screen min-h-screen bg-gray-900 flex flex-col items-center justify-center px-5 py-5">
+  <iframe
+    class="w-full md:w-1/4 h-56"
+    src="https://www.youtube.com/embed/z-FKAC0I_VQ"
+    title="EddieHub Community"
+    frameborder="0"
+    allow="accelerometer; clipboard-write; autoplay=1; encrypted-media; gyroscope; picture-in-picture"
+    allowfullscreen />
 
   <h3
```
Comment (kaiwalyakoparkar) [inline src/App.svelte:18]: You can make this in a line as well

File: src/App.svelte [hunk 3]
```diff
@@ -3,29 +3,28 @@
...
-    class="text-xl font-semibold text-indigo-100 leading-tight relative z-10"
-  >
+  <h2 class="text-xl font-semibold text-indigo-100 leading-tight relative z-10">
     OPEN SOURCE is NOT all about code, it is also the people and collaboration.
   </h2>
 
   <div
-    class="flex flex-col items-center bg-white lg:rounded m-12 py-8 px-4 lg:px-16 lg:max-w-4xl"
-  >
+    class="flex flex-col items-center bg-white lg:rounded m-12 py-8 px-4 lg:px-16 lg:max-w-4xl">
```
Comment (kaiwalyakoparkar) [inline src/App.svelte:27]: Make this in a line as well

File: src/App.svelte [hunk 4]
```diff
@@ -37,23 +36,27 @@
...
       {#each projects as cardItem}
         <Card {...cardItem} />
       {/each}
     </div>
   </div>
 
   <div
-    class="flex flex-col items-center bg-white lg:rounded m-12 py-8 px-4 lg:px-16 lg:max-w-4xl"
-  >
+    class="flex flex-col items-center bg-white lg:rounded m-12 py-8 px-4 lg:px-16 lg:max-w-4xl">
```
Comment (kaiwalyakoparkar) [inline src/App.svelte:59]: Can put this in a single line as well

File: src/App.svelte [hunk 5]
```diff
@@ -37,23 +36,27 @@
...
+
   <div id="projects">
     <h2 class="text-center text-3xl font-semibold text-indigo-100 mb-8 archivo">
       Our Projects
     </h2>
 
     <div
-      class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 xl:grid-cols-4 gap-4"
-    >
+      class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 xl:grid-cols-4 gap-4">
```
Comment (kaiwalyakoparkar) [inline src/App.svelte:51]: put this in single line as well

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]
If you are changing the coding style then please make sure you carry it out in the entire file. :smile: . I would recommend doing this small changes

**Comment by AllanRegush** [comment]
Sorry, I forgot to get back to this PR. Could you make sure the style matches up and only the changes you want to make are in the diff?

**Comment by pvcodes** [comment]
> Sorry, I forgot to get back to this PR. Could you make sure the style matches up and only the changes you want to make are in the diff?

For sure! the identation messed due to my vscode indenting setting, Sorry for that

**Comment by eddiejaoude** [comment]
Thank you for the contribution, but I will close this because website moved to low code platform

**Pull request closed without merging**